flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/batch/BatchExecExpand.scala

```diff
@@ -99,12 +99,12 @@ class BatchExecExpand(
       projects,
       opName = "BatchExpand")
 
-    new OneInputTransformation(
+    setManagedMemoryWeight(new OneInputTransformation(
```

Comment:
How about to introduce a util to create Transformations and set managed memory weight to 0 by default? So that the operators do not need to be aware of managed memory weight setting.